scripts/grace_sweep_report.py

```
@@ -0,0 +1,278 @@
+                net_cell = f"{merge_usd + exit_usd:>13.4f}"
+            else:
+                # Partial pricing is not a total. Withhold both.
+                exit_cell = f"{f'{priced_pairs}of{exposed_pairs}priced':>13}"
+                net_cell = f"{'unmeasured':>13}"
+            out.append(f"  {h:>7.0f}{merged_sh:>11.0f}{exposed_sh:>12.0f}"
+                       f"{merge_usd:>10.4f}{exit_cell}{net_cell}")
+        if ambiguous:
+            out.append("")
+            out.append(f"  ~ {ambiguous} pair-horizon case(s) had a companion arrive "
+                       f"within one poll of the horizon;")
+            out.append(f"    the poll phase is not recorded, so their merged/exposed "
```

Ruff (0.16.3)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)